Dockerfile-dev

```diff
@@ -0,0 +1,21 @@
+FROM node:16-alpine
+
+RUN apk update
+RUN apk add bash
+
+RUN npm i -g pnpm
+
+WORKDIR app
+
+COPY .npmrc package.json pnpm-lock.yaml ./
+RUN pnpm i -P --frozen-lockfile --shamefully-hoist --reporter=silent
+
+COPY config.dev.json webpack.server.config.js ./
+COPY static static
+COPY lib lib
+COPY build build
+COPY gun.db ./
```

Comment:
I don't really like copying a sqlite DB into a docker image.
But I don't know how to make it work otherwise.
Another reason to eventually move away from `gun` I guess...